staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -320,17 +320,15 @@ func (f *DeltaFIFO) queueActionLocked(actionType DeltaType, obj interface{}) err
 	newDeltas := append(f.items[id], Delta{actionType, obj})
 	newDeltas = dedupDeltas(newDeltas)
 
-	_, exists := f.items[id]
 	if len(newDeltas) > 0 {
-		if !exists {
+		if _, exists := f.items[id]; !exists {
```

Comment:
I probably wouldn't have made these changes at all but I guess I don't object :)

Reply:
(meaning, just these two--I'm a fan of the others)